### ISSUE TITLE
expo-av warning: `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.

### ISSUE THREAD

**Issue by belinde** [expo/expo#26770]
### Minimal reproducible example

https://github.com/belinde/expo-av-nativeeventemitter-bug

### Summary

The project has been generated as per standard procedure, then `expo-av` is added with `expo install`

```
npx create-expo-app expo-av-nativeeventemitter-bug
npx expo install expo-av
```

The relevant part of the code is simply:

```javascript
import { Audio } from "expo-av";
import { useCallback } from "react";
import { Button, StyleSheet, Text, View } from "react-native";

export default function App() {
  const playSound = useCallback(async () => {
    const sound = await Audio.Sound.createAsync(require("./assets/beep.mp3"));
    await sound.sound.playAsync();
  }, []);

  return (
    <View style={styles.container}>
      /* omissis */
      <Button title="Play Sound" onPress={playSound} />
    </View>
  );
}
/* omissis */
```

As long that the expo-av is `import`ed in the project, a couple of warnings keep appearing in the logs:

```
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.
```

Trying on android, with the latest available versions.

### Environment

```
  expo-env-info 1.2.0 environment info:
    System:
      OS: Linux 6.5 Ubuntu 23.10 23.10 (Mantic Minotaur)
      Shell: 5.2.15 - /bin/bash
    Binaries:
      Node: 20.10.0 - ~/.nvm/versions/node/v20.10.0/bin/node
      Yarn: 1.22.18 - ~/.yarn/bin/yarn
      npm: 10.2.5 - ~/.nvm/versions/node/v20.10.0/bin/npm
    npmPackages:
      expo: ~50.0.4 => 50.0.4 
      react: 18.2.0 => 18.2.0 
      react-native: 0.73.2 => 0.73.2 
    npmGlobalPackages:
      eas-cli: 7.1.1
      expo-cli: 6.3.10
    Expo Workflow: managed
```

**Comment by brentvatne** [comment]
hi there! is this in expo go?

**Comment by rogalski** [comment]
> hi there! is this in expo go?

This reproduces for me in Expo Go 2.30.9, with same `expo-av` in stacktrace.

**Comment by belinde** [comment]
> hi there! is this in expo go?

Yes, I've updated the bug description with the generation commands. I've noticed only now that the warnings are showing only in `--go` mode and not in `--dev-client`

**Comment by hamedasemi** [comment]
This error happens for me as well either way, it does not matter which flag is enabled

**Comment by CertainPassenger** [comment]
Same issue when using the `<Video>` component:

```javascript
<Video
  source={{
    uri: url,
  }}
/>
```

Packages:
"expo": "~50.0.4"
"expo-av": "~13.10.4"

**Comment by haikov** [comment]
> hi there! is this in expo go?

For me it's happening in the custom dev client as well and only on Android.

**Comment by hamedasemi** [comment]
For me, in expo go and only on Android

**Comment by VictorDev200** [comment]
For me, in expo go and Android. 

`
const sounds = {
    title: require("../assets/sounds/title.wav"),
}

export {
    sounds
}


import { Audio } from "expo-av";
import { sounds } from "../constants/Sounds";

export async function playSound(soundName:keyof typeof sounds, setSoundState:Function) {
  const { sound } = await Audio.Sound.createAsync(
    sounds[soundName]
  );

  setSoundState(sound);
  await sound.playAsync();
}
`

**Comment by brentvatne** [comment]
you will see this warning in expo go for now, until we release a new version that fixes it. outside of expo go, this is resolved. we'll likely release an update to expo go in the next week.

**Comment by alesanGreat** [comment]
I have attempted various workarounds, but it appears impossible to eliminate the warning on Android, preventing the app from running in Expo Go.

**Comment by LightPalmMedia** [comment]
Same issue here with "expo": "^50.0.7"  "expo-av": "~13.10.5",

**Comment by Luiyit** [comment]
I just noticed that for `expo-av` was fixed over the version `13.10.5`.
Thanks
